kbase-extension/kbase_templates/notebook.html

```diff
@@ -96,17 +96,15 @@
     </div>
     <div id="content-column">
         <span id="kb-add-code-cell"
-              class="kb-data-list-add-data-button fa fa-terminal fa-2x"
-              style="position:fixed; bottom:15px; right:75px; z-index:5"
+              class="fa fa-terminal fa-2x"
```

Comment:
The extra class, `kb-data-list-add-data-button`, was causing problems -- it is also used for the 'Add data' button, and some changes in that area had changed the positioning of these buttons as a result. I removed the class and used the element IDs as a hook for the styling.